crates/ra_hir/src/semantics.rs

```diff
@@ -271,6 +272,86 @@ impl<'db, DB: HirDatabase> Semantics<'db, DB> {
     pub fn assert_contains_node(&self, node: &SyntaxNode) {
         self.imp.assert_contains_node(node)
     }
+
+    pub fn is_unsafe_method_call(&self, method_call_expr: ast::MethodCallExpr) -> bool {
```

Comment:
These methods should be implemented on `imp`, to avoid duplicate monomorphisations.

Reply:
Done